CppImport/src/ClangHelpers.h

```diff
@@ -129,6 +129,19 @@ class CPPIMPORT_API ClangHelpers
 
 		void attachDeclarationComments(clang::NamedDecl* namedDecl, Model::Node* receiver) const;
 
+		/**
+		 * store the filename at location for node.
```

Comment:
This comment and the one for the method below are not very descriptive. What file name? how is a filename related to a location? Where are we storing it?